MolecularNodes/md.py

```diff
@@ -86,11 +86,34 @@ def load_trajectory(file_top,
     # separate the trajectory, separate to the topology or the subsequence selections
     traj = univ.trajectory[md_start:md_end:md_step]
     
+
     # if there is a non-blank selection, apply the selection text to the universe for 
     # later use. This also affects the trajectory, even though it has been separated earlier
     if selection != "":
```

Comment:
Does this involve using SolvationAnalysis? 

This is what I had in mind :

# define solvent AtomGroups
EA = univ.residues[0:235].atoms                    # ethyl acetate
FEC = univ.residues[235:600].atoms                 # fluorinated ethylene carbonate
PF6 = univ.atoms.select_atoms("byres element P")   # hexafluorophosphate

def test(selection):
    loc = {}
    exec(selection, globals(), loc)
    return loc

solvent_list= "EA = univ.residues[0:235].atoms,FEC = univ.residues[235:600].atoms"
list_solvent = solvent_list.split(",")
solv_dict={}
for i in list_solvent:
    solv_dict.update(test(i))


the solvent list (ONE Panel)


# instantiate solution
solute = Solute.from_atoms(li_atoms,
                    {'EA': EA, 'FEC': FEC, 'PF6': PF6},
                    radii={'PF6': 2.6, 'FEC': 2.7})

solute.run()

Reply:
Are you asking if that portion of the code should be removed for your PR?

Reply:
No, I'm giving an example of how to generalize this piece of code from the UI.  I want to make the lines of code string properties that are to be executed because Orion's Solute.from_atoms() dictionary format can't be accommodated in the UI . I am going to submit a PR demoing what I mean specifically. Haven't tested it out yet.

Reply:
I think the UI should take keys and values as strings and construct a dictionary